benchopt/cli/process_results.py

```diff
@@ -130,3 +130,30 @@ def generate_results(patterns=(), benchmarks=(), root=None, display=True):
     plot_benchmark_html_all(
         patterns=patterns, benchmarks=benchmarks, root=root, display=display
     )
+
+
+@process_results.command(
+    help="Convert a result file of type parquet into the requested type."
+)
+@click.argument('parquet_file',
+                default=Path.cwd(),
+                type=click.Path(exists=True))
+@click.option('--to', '-t', 'requested_formats',
+              metavar="<to>", multiple=True, type=str,
+              help="Choose the conversion format (CSV, JSON, Excel, etc.).")
+def convert(parquet_file, requested_formats):
```

Comment:
Not sure we need this.
It seems quite easy and it will be hard to test so not sure we want to maintain this?